### PR TITLE
Be more defensive about provided channelMax values

### PR DESCRIPTION
**Pull request by wangqiandeniangniang**
## Proposed Changes
1、format two line align
2、ChannelManager add channelMax < 0 condition, friendly tips

## Types of Changes

What types of changes does your code introduce to this project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes issue #NNNN)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (correction or otherwise)
- [x ] Cosmetics (whitespace, appearance)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating
the PR. If you're unsure about any of them, don't hesitate to ask on the
mailing list. We're here to help! This is simply a reminder of what we are
going to look for before merging your code._

- [x] I have read the `CONTRIBUTING.md` document
- [x] I have signed the CA (see https://cla.pivotal.io/sign/rabbitmq)
- [x] All tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in related repositories

## Further Comments

If this is a relatively large or complex change, kick off the discussion by
explaining why you chose the solution you did and what alternatives you
considered, etc.

### CODE REVIEW

File: src/main/java/com/rabbitmq/client/impl/ChannelManager.java [hunk 1]
```diff
@@ -70,6 +70,8 @@ public ChannelManager(ConsumerWorkService workService, int channelMax, ThreadFac
 
 
     public ChannelManager(ConsumerWorkService workService, int channelMax, ThreadFactory threadFactory, MetricsCollector metricsCollector) {
+        if (channelMax < 0)
+            throw new IllegalStateException("create ChannelManager: 'channelMax' must be greater or equal to 0.");
```
[the remark this answers is not in the document]
Comment (wangqiandeniangniang) [reply src/main/java/com/rabbitmq/client/impl/ChannelManager.java:74]: I consider the situation， channelMax  should be a positive integer， negative integer is illegal. Is there a better recommendation? thank you for reply.

[the remark this answers is not in the document]
Comment (michaelklishin) [reply src/main/java/com/rabbitmq/client/impl/ChannelManager.java:74]: The recommendation is to use `IllegalArgumentException` as a more appropriate exception type for invalid arguments.

[the remark this answers is not in the document]
Comment (wangqiandeniangniang) [reply src/main/java/com/rabbitmq/client/impl/ChannelManager.java:74]: Got it, Thanks

[the remark this answers is not in the document]
Comment (wangqiandeniangniang) [reply src/main/java/com/rabbitmq/client/impl/ChannelManager.java]: modified and submitted，trouble review code

**Comment by michaelklishin** [comment]
Thank you!

**Pull request merged**